app/assets/javascripts/my_modules/archived.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  let selectedTasks = [];
+
+  $('#module-archive').on('click', '.restore-button-container', function(e) {
+    e.preventDefault();
+    alert('woop! Not implemented yet!')
```

Comment:
What was that?

Reply:
It is a placeholder for someone who is doing on this feature. Not part of this task